resources/contribute-blog/_blog-template.md

```diff
@@ -0,0 +1,56 @@
+---
+category: 2025
+date: 2025-mm-dd
+seo:
+    title: SEO Optimized Title
+    description: SEO optimized description for the blog post that can be displayed in search results.
+
+labels:
+    - General
+markdown:
+    editPage:
+        hide: true
+---
+# SEO Optimized Title
+
+
+Congrats on taking the first step toward drafting your blog post! Use this template as a starting point to organize your blog post.
+
+<!-- BREAK -->
+
+Refer to [Contribute Blog](https://xrpl.org/resources/contribute-blog/) for detailed guidelines and examples.
+
+## Section: Handy Reference
+
+To help you get started quickly, here’s a handy guide to the syntax for some of the most frequently used components in a blog post — perfect for streamlining your writing process.
+
+Choosing an SEO optimized title helps boost your blog post. It is recommended to limit your title to under 50 characters and keep it same as the meta `seo -> title` field for consistency.
+
+To include an internal link, use a relative path.  For example, to create a link to the Contributing Documentation topic, use the following syntax: [Contribute Documentation](../resources/contribute-documentation/index.md).
+
+To include an external link, use the absolute URL instead of a relative path. For example, [Contribute Documentation](https://xrpl.org/resources/contribute-documentation).
+
+To include an unordered list, use the following syntax:
+
+- Item 1
+- Item 2
+
+To include an ordered list, use the following syntax:
+
+1. List item 1
+2. List item 2
+
+
+### Sub-section: Including Images In Your Blog Post
+
+To include an image, use the following syntax:
```

Comment:
Redundant instruction? Seems to be repeated on the next line.

Reply:
To include an image, do the following:

[_Store your graphics in the /blog/img directory_ isn't syntax, it's an instruction]